apps_test.go

```diff
@@ -25,7 +25,7 @@ func TestGetApps(t *testing.T) {
 
 func TestGetApp(t *testing.T) {
 	testServer := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
-		res.Write([]byte(`{"name": "alarm-clock"}`))
+		res.Write([]byte(`{"name": "alarm-clock","training_status":"done"}`))
```

Comment:
Please add here all fields which are possible to check that unmarshalling works well. Especially, please check if `time.Time` is converted correctly.